### PR TITLE
Re-enable BWB Pricing

### PR DESCRIPTION
**Pull request by mekarpeles**
Re-enables BWB pricing

### Technical

Disables bwb direct pricing, now only uses api

### Testing

Go to any edition page from sponsorship carousel, you should see price
https://dev.openlibrary.org/books/OL4258125M/Relax_win

### Stakeholders

@cdrini

### CODE REVIEW

**Comment by cdrini** [comment]
Stack:

```
TypeError: get_betterworldbooks_metadata() got an unexpected keyword argument 'thirdparty'
error saved to /var/log/openlibrary/ol-errors/2020-05-11/190357853831.html
Traceback (most recent call last):
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/infogami/utils/template.py", line 146, in saferender
    result = t(*a, **kw)
  File "/opt/openlibrary/venv/local/lib/python2.7/site-packages/web/template.py", line 881, in __call__
    return BaseTemplate.__call__(self, *a, **kw)
  File "/opt/openlibrary/venv/local/lib/python2.7/site-packages/web/template.py", line 808, in __call__
    return self.t(*a, **kw)
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/templates/type/edition/view.html", line 182, in __template__
    $for c in contributors:
  File "/opt/openlibrary/venv/local/lib/python2.7/site-packages/web/template.py", line 881, in __call__
    return BaseTemplate.__call__(self, *a, **kw)
  File "/opt/openlibrary/venv/local/lib/python2.7/site-packages/web/template.py", line 808, in __call__
    return self.t(*a, **kw)
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/macros/LoanStatus.html", line 88, in __template__
    class="cta-btn cta-btn--sponsor"
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/core/sponsorships.py", line 144, in qualifies_for_sponsorship
    bwb_price = get_betterworldbooks_metadata(edition.isbn, thirdparty=True).get('price_amt')
TypeError: get_betterworldbooks_metadata() got an unexpected keyword argument 'thirdparty'
```

**Comment by mekarpeles** [comment]
Resolved: https://github.com/internetarchive/openlibrary/pull/3450/files#diff-7e798ee43729cebf62d36874700ada7eR144

File: openlibrary/macros/databarWork.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 $# Disable price temporarily: NEL causing too much traffic to BWB!
 $# Was: page.key.startswith('/books/')
-$ prices = False
+$ prices = True
```
Comment (cdrini) [inline openlibrary/macros/databarWork.html:14]: What should this be? The comment suggests it should be something else.

Comment (cdrini) [reply openlibrary/macros/databarWork.html:14]: Also remove the comment

Comment (cdrini) [reply openlibrary/macros/databarWork.html]: fixed

**Pull request merged**